katdal/vis_flags_weights.py

```diff
@@ -39,25 +39,32 @@
 class VisFlagsWeights(object):
     """Correlator data in the form of visibilities, flags and weights.
 
+    This container stores the triplet of visibilities, flags and weights
+    as dask arrays to provide lazy / deferred access to the data.
```

Comment:
Except it actually stores a quadruple.

Reply:
😄 

I was quoting [CASA](https://casacore.github.io/casacore-notes/264.pdf), which calls it the (DATA, WEIGHT, FLAG) triplet. For me, unscaled weights is a derived product of vis and weights, so not really a fourth member.